src/span_context.js

```diff
@@ -134,7 +134,7 @@ export default class SpanContext {
     this._spanIdStr = null;
   }
 
-  set parentId(parentId: Buffer): void {
+  set parentId(parentId: Buffer | null): void {
```

Comment:
what does `| null` mean? It should not be legal to pass null to this method

Reply:
It is necessary because parentId is set to null here: https://github.com/jaegertracing/jaeger-client-node/blob/master/src/tracer.js#L226

Reply:
When doing `ctx.parentId = null`, this set function is called. [From MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set): `The set syntax binds an object property to a function to be called when there is an attempt to set that property.` 

Thus, as it is written currently, this setter is called with null. Flow understands this and triggers an error. Here is a small example: https://flow.org/try/#0PTAEAEDMBsHsHcBQiDG0CGBnToAqBTTAFwGENtQBvRUUTfI0AB3QCd8A7IgSQBNQAFAEteALjpFWQjgHMAlFRq1QRABZDMAOhbsufUAF5QIgNy0lAX0RXUsDsRWFGRjvnh4nZLJgFzkRJ202Th5+IwAiHRC+cMQA4iDdUMNQDgBXaGggA

Reply:
@TLadd Thanks for upgrading flow! Awesome!

Just curious, why not mark the field as `?Buffer` ?

Reply:
It's not particularly important in this case either way. ?Buffer is equivalent to `Buffer | null | void`. So by specifying `parentId: Buffer | null`, it is not legal to do `ctx.parentId = undefined` whereas that would be fine with ?Buffer. Since we only had the one case of setting parentId to null, I figured not allowing undefined was reasonable.